packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -802,7 +801,7 @@ Future<bool> disableAnalytics() async {
 /// devtoolsChrome.
 void computeDevToolsCustomGTagsData() {
   // Platform
-  final String platform = window.navigator.platform!;
+  final String platform = window.navigator.platform;
   platform.replaceAll(' ', '_');
```

Comment:
This is a NO-OP, btw. `replaceAll` returns a `String`!!

@kenzieschmoll

Reply:
Noted! Let's leave for now since we are switching to unified_analytics soon.